Using expresssions (#19)
opulencephp/docs: database-query-builders.md

> **Note:** `INSERT` and `UPDATE` query builders bind unnamed placeholder values.  To specify the type of the value, use an array whose first item is the value and whose second item is the type.

<h3 id="using-expressions-in-insert-queries">Using Expressions In Insert Queries</h3>

> If you need to set some values as expressions, the column names to values array can contain `Expression` objects to help with that:

Reviewer: ```suggestion
If you need to set some values as expressions, you can use `Expression` objects in the column names to values array:
```

```php
use Opulence\QueryBuilders\QueryBuilder;

> use Opulence\QueryBuilders\Expression;

Reviewer: Can you alphabetize the `use` statements?

> **Note:** Like `INSERT` query builders, `UPDATE` query builders bind unnamed placeholder values.

<h3 id="using-expressions-in-update-queries">Using Expressions in Update Queries</h3>

> Using expressions to set value in `UPDATE` queries work exactly the same as for [INSERT Queries](#using-expressions-in-insert-queries).

Reviewer: ```suggestion
You can use SQL expressions in `UPDATE` queries just like you can with [`INSERT` queries](#using-expressions-in-insert-queries).
```
